### PR TITLE
🐛 Remove unneeded caBundle: Cg== value

### PR DESCRIPTION
**Pull request by matthchr**
- As per the comment in source, this field is no longer needed.

### CODE REVIEW

**Comment by matthchr** [comment]
Related to https://github.com/kubernetes-sigs/kubebuilder/pull/1699

**Comment by matthchr** [comment]
@DirectXMan12 Can you take a look at this?

File: pkg/crd/markers/zz_generated.markerhelp.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ func (Minimum) Help() *markers.DefinitionHelp {
 	return &markers.DefinitionHelp{
 		Category: "CRD validation",
 		DetailedHelp: markers.DetailedHelp{
-			Summary: "specifies the minimum numeric value that this field can have.",
+			Summary: "specifies the minimum numeric value that this field can have. Negative integers are supported.",
```
Comment (matthchr) [inline pkg/crd/markers/zz_generated.markerhelp.go:201]: Note: I didn't add this... maybe somebody forgot to regenerate something?

**Comment by matthchr** [comment]
@DirectXMan12 @pwittrock - Can you take a look?

File: pkg/crd/markers/zz_generated.markerhelp.go (hunk 1 above)
Comment (DirectXMan12) [reply pkg/crd/markers/zz_generated.markerhelp.go:201]: yeah, someone forgot to regenerate, and the testing missed it somehow.

**Pull request merged**